QuizApp/Custom/CustomRoundButton.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  CustomRoundButton.swift
+//  QuizApp
+//
+//  Created by Ana on 11/14/24.
+//
+
+import UIKit
+
+final class CustomRoundButton: UIButton {
+    
+    convenience init() {
+        self.init(frame: .zero)
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupButton()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupButton() {
+        backgroundColor = Constants.Colors.yellowPrimary
+        layer.cornerRadius = 22.5
+        translatesAutoresizingMaskIntoConstraints = false
+        
+//        NSLayoutConstraint.activate([
+//            widthAnchor.constraint(equalToConstant: 45),
+//            heightAnchor.constraint(equalToConstant: 45)
+//        ])
+    }
+    
+    func configure(with image: UIImage?) {
```

Comment:
Move every default value that isn't being changed into Constraints and constant enum created in the same file, not only in this function but above in "setupButton" method as well